core/src/main/scala/kafka/utils/Log4jController.scala

```diff
@@ -17,19 +17,10 @@
 
 package kafka.utils
 
-
-import org.apache.log4j.{Logger, Level, LogManager}
 import java.util
 import java.util.Locale
 
-
-object Log4jController {
```

Comment:
We seem to have removed the companion object but didn't reindent the now top level class?

Reply:
Hmm, could you please elaborate on this? Looking at the code it seems to be correct, no checkstyle failures, no unnecessary empty lines.

Reply:
My bad, seems like I misread the diff. I should point out that checkstyle doesn't run on Scala code though.

Reply:
No problem, and it's good to know that it doesn't run on Scala.

Reply:
Yes.